### PR TITLE
fix(example/display_rotation): Enable PSRAM for LCD_EV board

### PR DESCRIPTION
**Pull request by tore-espressif**
# Display rotation on LCD_EV board

The LCD_EV board must have PSRAM enabled for all its frame buffers.

But even with PSRAM enabled, the image is distorted:

@Lzw655 Could you please have a look?

Potentially blocks https://github.com/espressif/esp-bsp/pull/221 (@XDanielPaul if this issue is not resolved quickly we can omit it from intial LaunchPad release)

### CODE REVIEW

**Comment by tore-espressif** [comment]
@Lzw655 Can I assign this to you?

File: examples/display/sdkconfig.bsp.esp32_s3_lcd_ev_board [hunk 1]
```diff
@@ -4,6 +4,8 @@
 CONFIG_IDF_TARGET="esp32s3"
 CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
 CONFIG_ESP_DEFAULT_CPU_FREQ_240=y
+CONFIG_SPIRAM=y
+CONFIG_SPIRAM_MODE_OCT=y
 CONFIG_LV_COLOR_16_SWAP=y
 CONFIG_LV_MEM_CUSTOM=y
 CONFIG_LV_MEMCPY_MEMSET_STD=y
```
Comment (Lzw655) [inline examples/display/sdkconfig.bsp.esp32_s3_lcd_ev_board:11]:
```suggestion
CONFIG_ESPTOOLPY_FLASHMODE_QIO=y  
CONFIG_COMPILER_OPTIMIZATION_PERF=y 
CONFIG_SPIRAM=y 
CONFIG_SPIRAM_MODE_OCT=y
CONFIG_SPIRAM_FETCH_INSTRUCTIONS=y 
CONFIG_SPIRAM_RODATA=y 
CONFIG_SPIRAM_SPEED_80M=y
CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240=y 
CONFIG_FREERTOS_HZ=1000 
CONFIG_LV_MEM_CUSTOM=y 
CONFIG_LV_MEMCPY_MEMSET_STD=y 
CONFIG_LV_ATTRIBUTE_FAST_MEM_USE_IRAM=y
```

File: examples/display_rotation/sdkconfig.bsp.esp32_s3_lcd_ev_board [hunk 2]
```diff
@@ -6,3 +6,5 @@ CONFIG_LV_COLOR_16_SWAP=y
 CONFIG_LV_MEM_CUSTOM=y
 CONFIG_LV_MEMCPY_MEMSET_STD=y
 CONFIG_LV_USE_PERF_MONITOR=y
+CONFIG_SPIRAM=y
+CONFIG_SPIRAM_MODE_OCT=y
```
Comment (Lzw655) [inline examples/display_rotation/sdkconfig.bsp.esp32_s3_lcd_ev_board:10]:
```suggestion
CONFIG_ESPTOOLPY_FLASHMODE_QIO=y 
CONFIG_COMPILER_OPTIMIZATION_PERF=y 
CONFIG_SPIRAM=y CONFIG_SPIRAM_MODE_OCT=y
CONFIG_SPIRAM_FETCH_INSTRUCTIONS=y 
CONFIG_SPIRAM_RODATA=y 
CONFIG_SPIRAM_SPEED_80M=y
CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240=y 
CONFIG_FREERTOS_HZ=1000 
CONFIG_LV_MEM_CUSTOM=y 
CONFIG_LV_MEMCPY_MEMSET_STD=y 
CONFIG_LV_USE_PERF_MONITOR=y
CONFIG_LV_ATTRIBUTE_FAST_MEM_USE_IRAM=y
```

**Review by Lzw655 — COMMENTED** [review]
Sorry, I'm in a vacation. Please try to use these configurations and you can discard some of them except the following configurations. Otherwise, you may explore screen drift.
``` CONFIG_COMPILER_OPTIMIZATION_PERF=y 
CONFIG_SPIRAM=y 
CONFIG_SPIRAM_MODE_OCT=y
CONFIG_SPIRAM_FETCH_INSTRUCTIONS=y 
CONFIG_SPIRAM_RODATA=y 
CONFIG_SPIRAM_SPEED_80M=y
CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240=y 
CONFIG_FREERTOS_HZ=1000 
````

**Comment by Lzw655** [comment]
BTW, you can find out the reasons and solutions about screen drift from [docs](https://docs.espressif.com/projects/espressif-esp-faq/en/latest/software-framework/peripherals/lcd.html#why-do-i-get-drift-overall-drift-of-the-display-when-driving-an-rgb-lcd-screen).

**Comment by tore-espressif** [comment]
> Sorry, I'm in a vacation. Please try to use these configurations and you can discard some of them except the following configurations. Otherwise, you may explore screen drift.
> 
> ```
> CONFIG_SPIRAM=y 
> CONFIG_SPIRAM_MODE_OCT=y
> CONFIG_SPIRAM_FETCH_INSTRUCTIONS=y 
> CONFIG_SPIRAM_RODATA=y 
> CONFIG_SPIRAM_SPEED_80M=y
> CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240=y 
> CONFIG_FREERTOS_HZ=1000 
> ```

Thank you, that helped!

**Review by espzav — APPROVED** [review]
@tore-espressif and @Lzw655 Thank you for this fix. All tested on HW. LGTM

**Comment by tore-espressif** [comment]
Fixed by https://github.com/espressif/esp-bsp/pull/225

**Pull request closed without merging**